spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -115,8 +115,8 @@
                                   cpu: biggest_os_flavor.cpu,
                                   hdd: biggest_os_flavor.hdd,
                                   tenant: amazon)
-          biggest_os_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 100)
-          optimal_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 100)
+          biggest_os_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 1)
+          optimal_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 1)
```

Comment:
Can you point me to the place where amazon flavors are created? I'm trying to understand this spec, but it is quite complicated. I come into conclusion that to complicated factories are baaaaaaaaaaad, so many unclear dependencies and assumptions :cry: